sdk/cognitiveservices/Vision.Face.Preview/README.md

```diff
@@ -0,0 +1,55 @@
+# Azure Cognitive Services Face preview client library for .NET
+
+## Getting started
+
+### Install the package
+
+Install the Face client library for .NET with [NuGet][nuget]:
+
+```PowerShell
+dotnet add package Microsoft.Azure.CognitiveServices.Vision.Face.Preview --version  2.7.0-preview.2
+```
+
+### Prerequisites
+
+* An [Azure subscription][azure_sub].
+* An existing Cognitive Services or Face resource.
+
+### Authenticate the client
+
+To authenticate the FaceClient, you will need to provide the endpoint and API key for your Face resource.
+You can obtain the endpoint and API key from the resource information in the [Azure Portal][azure_portal].
+Instantiate the FaceClient with ApiKeyServiceClientCredentials as follows:
+
+```
```

Comment:
We'll want to make this a snippet so that we can be sure that it gets compiled nightly to avoid stale and outdated documentation.   
_(see: [Updating Sample Snippets](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#updating-sample-snippets))_